iis-wf/iis-wf-referenceextraction/src/test/resources/eu/dnetlib/iis/wf/referenceextraction/covid19/main/sampletest_empty_input/oozie_app/workflow.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0"?>
```

Comment:
1. I'd remove all comments in this file, matching comments are in `iis-examples`
1. formatting is a bit wrong, so I'd use IDE's auto formatting feature to fix it